### DISCUSSION TITLE
Welcome to hexagonal-architecture Discussions!

### DISCUSSION THREAD

**Discussion by yinebebt** [Yinebeb-01/hexagonal-architecture#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Dru-Go** [comment]
Nice work, I would love to contribute, create more issues you like to work on, and perhaps I would go at it...

**Reply by yinebebt** [reply]
let us collaborate  . . .

**Reply by Dru-Go** [reply]
Yes please
